src/test/java/kitchenpos/application/OrderTableServiceTest.java

```diff
@@ -0,0 +1,186 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.*;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static kitchenpos.application.OrderFixture.createOrderRequest;
+import static kitchenpos.application.OrderTableFixture.createOrderTableRequest;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.only;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class OrderTableServiceTest {
+
+    @Mock
+    private OrderTableRepository orderTableRepository;
+
+    @Mock
+    private OrderRepository orderRepository;
+
+    @InjectMocks
+    private OrderTableService orderTableService;
+
+    @BeforeEach
+    void setUp() {
+        orderTableService = new OrderTableService(orderTableRepository, orderRepository);
+    }
+
+    @DisplayName("테이블을 생성할 수 있다.")
+    @ParameterizedTest
+    @CsvSource(value = {"테이블1, 4"})
+    void create(final String name, final int numberOfGuests) {
+        //given
+        final OrderTable requestTable = createOrderTableRequest(name, numberOfGuests);
+        when(orderTableRepository.save(any(OrderTable.class))).thenReturn(requestTable);
+
+        //when
+        final OrderTable actualRTable = orderTableService.create(requestTable);
```

Comment:
```suggestion
        //when
        when(orderTableRepository.save(any(OrderTable.class))).thenReturn(requestTable);
        final OrderTable actualRTable = orderTableService.create(requestTable);
```

when 이 있는데 // when 영역을 표현하게 되면 헷갈리 수 있을 것 같아요!
이와 표현해보면 좋을 것 같아요!

>https://velog.io/@lxxjn0/Mockito%EC%99%80-BDDMockito%EB%8A%94-%EB%AD%90%EA%B0%80-%EB%8B%A4%EB%A5%BC%EA%B9%8C

이 글을 읽어보고 given 을 활용해 볼 수도 있을 것 같아요!